daemon/build.go

```diff
@@ -75,15 +76,24 @@ func (rl *releaseableLayer) Release() error {
 	if rl.released {
 		return nil
 	}
+	if err := rl.releaseRWLayer(); err != nil {
+		return err
+	}
+	if err := rl.releaseROLayer(); err != nil {
```

Comment:
It's probably better to try releasing `ro` even if `rw` fails. It really shouldn't happen but if this function fails there is probably a data leak that will not get cleaned up before daemon restart.